### PR TITLE
Added support to run component locally

### PR DESCRIPTION
**Pull request by Josca**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -42,19 +42,38 @@ services:
...
 
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.4.0
+    container_name: zookeeper
+    env_file:
+      - ./conf/zookeeper.env
+    ports:
+      - 2181:2181
+
+  kafka:
```
Comment (semtexzv) [inline docker-compose.yml:53]: Couldn't we run an image with contains both zookeper and kafka ?

File: platform/platform.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ import (
 var websockets []chan string
 var runUploadLoop = false
 
-func AddWebsocket() chan string {
+func addWebsocket() chan string {
```
Comment (semtexzv) [inline platform/platform.go:17]: Why are all these public ? We are calling them only in the `platform` package.

File: docker-compose.yml (hunk 1 above)
Comment (Josca) [reply docker-compose.yml:53]: Yes, but we would have to build it ourself. This is standard way according to kafka docs.

File: platform/platform.go (hunk 2 above)
Comment (Josca) [reply platform/platform.go:17]: This is question for you, you wrote this code. I made it private right in this PR.

**Pull request merged**